### PR TITLE
AFK notifier workflow "as code"

### PR DESCRIPTION
**Pull request by psrpinto**
This PR implements the AFK notifier workflow, without using engine, aka "as code".

### CODE REVIEW

File: resources/workflows.toml [hunk 1]
```diff
@@ -24,11 +24,10 @@
 #################################################################
 
 [[workflow]]
-identifier = "A8C-AFK"
+identifier = "afk_notifier"
```
Comment (psrpinto) [inline resources/workflows.toml:27]: As discussed, we'll reserve `afk` for a possible future `!afk` command.

File: app/app.go [hunk 2]
```diff
@@ -60,6 +61,12 @@ func (app app) runWorkflow(workflow w.Workflow, payload map[string]string) error
 	switch workflow.Identifier {
 	default:
 		runner = app.engine
+	case "afk_notifier":
+		matrixClient, err := app.botRegistry.GetClient("afk")
```
Comment (ashfame) [inline app/app.go:65]: Was this identifier where we agreed to use bot usernames?

**Review by ashfame — APPROVED** [review]
Looks good 🚀

File: app/app.go (hunk 2 above)
Comment (psrpinto) [reply app/app.go:65]: Yep, indeed, I opened https://github.com/Automattic/neurobot/issues/73 to track this, and will take care of it swiftly.

**Pull request merged**